plugins/com.gwtplugins.gdt.eclipse.suite/src/com/google/gdt/eclipse/suite/wizards/NewWebAppTemplateProjectWizard.java

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ * Copyright 2011 Google Inc. All Rights Reserved.
```

Comment:
new files, should have updated copyright date, and owner

Reply:
Fixed this